### PR TITLE
Prettify MD/YAML files

### PR DESCRIPTION
**Pull request by kianmeng**
Resolved via `prettier -w .`

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -1,95 +1,115 @@
...
 
 Input
+
 - [ ] Password input (#58) (https://github.com/conradkdotcom/rpassword) (https://github.com/antirez/linenoise/issues/125)
-- [X] quoted insert (#65)
+- [x] quoted insert (#65)
 - [ ] Overwrite mode (em-toggle-overwrite, vi-replace-mode, rl_insert_mode)
 - [ ] Encoding
-- [X] [Ctrl-][Alt-][Shift-]<Key> (#121)
+- [x] [Ctrl-][alt-][Shift-]<Key> (#121)
```
Comment (gwenn) [inline TODO.md:62]: Why `alt` ?

File: README.md [hunk 2]
```diff
@@ -205,26 +210,26 @@ $ infocmp
...
+| [Liner][]          | Rust    | Ux     | ANSI |         | No inc search | only word  | Emacs/vi/prog | No        | Yes  | Ux         | History based     |
+| [prompt_toolkit][] | Python  | Ux/Win | ANSI | Yes     | Yes           | any        | Emacs/vi/conf | Yes       | Yes  | Ux/Win     | Yes               |
+| [reedline][]       | Rust    | Ux/Win | ANSI | Yes     | Yes           | any        | Emacs/vi/bind | No        | Yes  | Ux/Win     | Yes               |
+| [replxx][]         | C/C++   | Ux/Win | ANSI | Yes     | Yes           | only line  | Emacs         | Yes       | No   | Ux/Win     | Yes               |
+| Rustyline          | Rust    | Ux/Win | ANSI | Yes     | Yes           | any        | Emacs/vi/bind | Yes       | Yes  | Ux/Win 10+ | Yes               |
+| [termwiz][]        | Rust    | Ux/Win | Any  | ?       | Yes           | any        | Emacs         | No        | No   | Ux/Win     | No                |
 
 [go-prompt]: https://github.com/c-bata/go-prompt
-[Haskeline]: https://github.com/judah/haskeline
+[haskeline]: https://github.com/judah/haskeline
```
Comment (gwenn) [inline README.md:228]: TODO check that link is not broken

File: Incremental.md [hunk 3]
```diff
@@ -2,26 +2,29 @@
...
+- line(s) buffer,
+- cursor position,
+- hint / input validation message,
+- screen size (line wrapping),
+- [prompt continuation](https://github.com/kkawakam/rustyline/pull/372)s,
+- row/wrap count.
 
 Some other values may impact layout but they are/should be constant:
-  * tab stop,
+
```
Comment (gwenn) [inline Incremental.md:26]: Not sure the extra line is prettier...

Comment (kianmeng) [reply Incremental.md:26]: It's related to [Incremental.md:26: MD032 Lists should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md032---lists-should-be-surrounded-by-blank-lines).

File: TODO.md (hunk 1 above)
Comment (kianmeng) [reply TODO.md]: Probably a bug with `prettier`. Reverted this.

**Comment by gwenn** [comment]
Thanks.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**